tests/integration/multimodal/test_multimodal_parallel.py

```diff
@@ -80,8 +80,9 @@ def multimodal_all_types_documents():
     return docs
 
 
-def test_multimodal_all_types_parallel(multimodal_all_types_documents):
+def test_multimodal_all_types_parallel(multimodal_all_types_documents, is_callback_called):
     def validate_response(resp):
+        is_callback_called._callback_called = True
```

Comment:
I don't see, why this should be better than `global`. I believe `global` itself is better, since it is explicit and not implicit.